src/sensors.c

```diff
@@ -64,9 +64,7 @@ static char *sensor_type_name_map[] =
 	"temperature",
 # define SENSOR_TYPE_POWER       3
 	"power",
-# define SENSOR_TYPE_CURRENT     4
-	"current",
-# define SENSOR_TYPE_UNKNOWN     5
```

Comment:
But you are using SENSOR_TYPE_CURRENT unguarded in known_features[]. If it is not defined, the code breaks. Please check your code against an older version of libsensors.